kwok/Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM registry.k8s.io/kwok/cluster:v0.4.0-k8s.v1.27.3
 
-ARG CLUSTER_NAME=kwok
-ENV CLUSTER_NAME=${CLUSTER_NAME}
+ENV CLUSTER_NAME=kwok
+ENV MEMBER1_CLUSTER_NAME=kwok-m1
+ENV MEMBER2_CLUSTER_NAME=kwok-m2
```

Comment:
I think we might as well call these "host", "m01" and "rh01" to make the names familiar to people who know about the RHTAP clusters.

Also I don't see a benefit for using env vars here as opposed to the explicit names.

Reply:
See whether you like it better now.